pkg/core/connection/api/defaultapi/default_client.go

```diff
@@ -0,0 +1,278 @@
+package defaultapi
```

Comment:
Do we know why this appears as new file?

Reply:
Because it is a new file. Create package separation between interface and implementation to prevent circular dependency. Maybe there is a better way.